configs/components/ruby-3.2.0.rb

```diff
@@ -64,11 +64,9 @@
   end
 
   if platform.is_windows?
- #   pkg.apply_patch "#{base}/windows_ruby_2.5_fixup_generated_batch_files.patch"
- #   pkg.apply_patch "#{base}/windows_nocodepage_utf8_fallback_r2.5.patch"
- #   pkg.apply_patch "#{base}/win32_long_paths_support.patch"
+    pkg.apply_patch "#{base}/windows_mingw32_mkmf.patch"
+    pkg.apply_patch "#{base}/windows_nocodepage_utf8_fallback_r2.5.patch"
  #   pkg.apply_patch "#{base}/ruby-faster-load_27.patch"
```

Comment:
Need to confirm if this patch is still needed in https://github.com/ruby/ruby/blob/88e0862267cec6764ab9888f7522a40f2996e2c7/load.c#L690-L691

Reply:
I filed this as https://tickets.puppetlabs.com/browse/PA-4911 as it'll be easier to do perf testing once we have puppet-agent builds with ruby 3.2 in them.